conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-input-mouse \
+            xf86-video-fbdev \
+            xf86-input-keyboard"
```

Comment:
Like 'mouse', 'keyboard' will be ignored when using 'evdev'. And if you insist on keeping them, can you at least group input drivers and ddx's together?